framework/src/media/audio/audio_manager.c

```diff
@@ -570,6 +641,10 @@ audio_manager_result_t set_audio_stream_in(uint8_t channels, uint32_t sample_rat
 
 	medvdbg("[IN] Device samplerate: %u, User requested: %u\n", config.rate, sample_rate);
 
+	if ((ret = audio_manager_mutex_lock(INPUT)) != AUDIO_MANAGER_SUCCESS) {
```

Comment:
As I mentioned line 358, below can be basic operation
```
set_audio_stream_in
{
    if (g_actual_audio_in_card_id == INVALID) {
          return AUDIO_MANAGER_CARD_NOT_FOUND;
    }
    ....
    pthread_mutex_lock(g_audio_in_cards[g_actual_audio_in_card_id].mutex);
```

Reply:
Modified in each stream function.